src/stats/boxplot.jl

```diff
@@ -75,7 +75,6 @@ _flip_xy(r::Rect{2,T}) where {T} = Rect{2,T}(reverse(r.origin), reverse(r.widths
 
 function Makie.plot!(plot::BoxPlot)
     args = @extract plot (width, range, show_outliers, whiskerwidth, show_notch, orientation, x_gap, dodge, n_dodge, dodge_gap)
-
```

Comment:
Here and below, maybe it's better to not change white space.

Reply:
They were artifacts left from me messing around. They are removed.